CommentingPractice/comment-practice.js

```diff
@@ -105,6 +105,9 @@ import ColorConverter from './services/colorConverter';
 
     member.hello();
```

Comment:
The class "TeamMate" doesn't seem to have a "hello()" method, so the program would crash here.